openstreetmap-install.sh

```diff
@@ -36,10 +36,16 @@ sudo apt install -y postgresql-server-dev-$pg_version
 # Installing pgxnclient
 sudo apt install -y pgxnclient
 ## pgxn has to call pg_config, add PG pinaries to PATH
-sudo echo 'export PATH=$PATH:/usr/local/pgsql/bin' >> /root/.bashrc
+## sudo echo 'export PATH=$PATH:/usr/local/pgsql/bin' >> /root/.bashrc
```

Comment:
if this code should not be executed then you need to remove it. there is git history if you will want to find this code in the future.

Reply:
removed